bin/asinit

```diff
@@ -8,6 +8,30 @@ const colors = require("../cli/util/colors");
 const version = require("../package.json").version;
 const options = require("../cli/util/options");
 
+const commands = {
+  "npm": {
+    install: "npm install",
+    run: "npm run",
+    test: "npm test"
+  },
+  "yarn": {
+    install: "yarn install",
+    run: "yarn",
+    test: "yarn test"
+  },
+  "pnpm": {
+    install: "pnpm install",
+    run: "pnpm run",
+    test: "pnpm test"
+  }
+};
+
+let pm = "npm";
+if (process.env.npm_config_user_agent.match(/\byarn\//))
```

Comment:
nice! So if user wrote `yarn asinit .` it will create template project configured for yarn pm, right?

Reply:
Yep

Reply:
```suggestion
if (process.env.npm_config_user_agent.test(/\byarn\//))
```